README.md

```diff
@@ -75,3 +75,68 @@ This project is still in early stages but future (exciting!) plans include:
 [issue94]: https://github.com/jkcfg/jk/issues/94
 [issue110]: https://github.com/jkcfg/jk/issues/110
 [issueHermeticity]: https://github.com/jkcfg/jk/issues?q=is%3Aissue+is%3Aopen+label%3Atopic%2Fhermeticity
+
+## Contributing
+When contributing to this repository, please first discuss the change you wish to make via issue with the owners of this repository before making a change.
```

Comment:
If we want this (@dlespiau - how do you feel about this? maybe just for bigger changes?), maybe add a link to Slack (or which other way of communication?).

Reply:
Maybe, but let's make this a separate change, we can do it later with the self-invite link (fwiw, there's a #jkcfg channel on the weave community slack).

Reply:
Remove or leave?

Reply:
leave what you wrote :) we'll add a link to a slack channel in another PR.